tests/InputFiles/NewtonianEuler/RiemannProblem3D.yaml

```diff
@@ -52,5 +52,5 @@ EventsAndTriggers:
   : - Completion
 
 Observers:
-  VolumeFileName: "./NewtonianEulerVolume"
-  ReductionFileName: "./NewtonianEulerReductions"
+  VolumeFileName: "./NewtonianEulerVolume3D"
```

Comment:
Here too, probably better to include the name: `NewtonianEulerRiemannVolume3D` (same for reduction)